releasenotes/notes/deprecation_0.9-0163a90875b53832.yaml

```diff
@@ -0,0 +1,6 @@
+---
+upgrade:
+  - |
```

Comment:
What upgrade entries for: `Basis set as plain string.` and `as_dict in the base validator`?

Reply:
Done in 0787dd910